doc/user_doc/vic_installation/vch_installer_examples.md

```diff
@@ -0,0 +1,193 @@
+# Examples of Deploying a Virtual Container Host #
+
+This topic provides examples of the `vic-machine` options to use when deploying virtual container hosts in different vSphere configurations.
+
+- [Deploy a Virtual Container Host Directly on an ESXi Host with no Resource Pools](#esxi)
+- [Deploy a Virtual Container Host in a vCenter Server Cluster](#cluster)
+- [Deploy a Virtual Container Host to a Cluster and Specify External, Management, and Container Networks](#networks)
+- [Deploy a Virtual Container Host and Specify a Different Datastore for Container VM Files](#datastores)
+- [Deploy a Virtual Container Host on a Standalone Host in vCenter Server](#standalone)
+- [Deploy a Virtual Container Host in a Resource Pool on an ESXi Host](#rp_host)
+- [Deploy a Virtual Container Host in a Resource Pool in a vCenter Server Cluster](#rp_cluster)
+- [Deploy a Virtual Container Host for Use with a Docker Client that Requires CA Certificates](#ca_cert)
+- [Deploy a Virtual Container Host with no TLS Authentication](#no_tls)
+- [Deploy a Virtual Container Host with Customized VM Settings](#customized)
+
+<a name="esxi"></a>
+## Deploy a Virtual Container Host Directly on an ESXi Host with no Resource Pools##
+
+You can install vSphere Integrated Containers directly on an ESXi host that is not managed by a vCenter Server instance. This example provides the miniumum options required to install vSphere Integrated Containers. The installer prompts you for the password for the ESXi host and deploys a virtual container host appliance with the default name `docker-appliance`. When an ESXi host has no resource pools, you must provide the address of  the root resource pool in the `compute-resource` option in the `govc` format, exactly as shown in the example. You specify the datastore in which to store the virtual container host files and container image files in the mandatory `image-store` option.
```

Comment:
Yes, that is what I meant. It would be a nice to have for GA and something to look at post 1.0.